apps/server/src/modules/tool/uc/school-external-tool.uc.ts

```diff
@@ -35,8 +35,9 @@ export class SchoolExternalToolUc {
 
 		await this.schoolExternalToolValidationService.validateCreate(schoolExternalTool);
 
-		const createdSchoolExternalTool: SchoolExternalToolDO =
-			await this.schoolExternalToolService.createSchoolExternalTool(schoolExternalTool);
+		const createdSchoolExternalTool: SchoolExternalToolDO = await this.schoolExternalToolService.saveSchoolExternalTool(
```

Comment:
I think we generally don't state the type of variables but I'm not sure if we have a guideline on this. I've just hardly ever seen it in our code.

Reply:
There are no guidelines yet